test/integration/features/cmd-timezone.feature

```diff
@@ -0,0 +1,72 @@
+@cmd-timezone @core
+Feature: Minishift timezone subcommand
+Timezone subcommand is used for listing and applying timezone 
+availaible to Minishift running VM instance.
+
+@quick
+Scenario: User cannot run timezone subcommand without a running Minishift instance
+  Given Minishift has state "Does Not Exist"
+   When executing "minishift timezone" succeeds
+    And stdout should contain
+    """
+    Running this command requires an existing 'minishift' VM
+    """
+
+@quick
+Scenario: User cannot run timezone flag --list without a running Minishift instance
+  Given Minishift has state "Does Not Exist"
+   When executing "minishift timezone --list" succeeds
+    And stdout should contain
+    """
+    Running this command requires an existing 'minishift' VM
+    """
+
+@quick
+Scenario: User cannot set a specific timezone without a running Minishift instance
+  Given Minishift has state "Does Not Exist"
+   When executing "minishift timezone --set Asia/Kolkata" succeeds
+   Then stdout should contain
+    """
+    Running this command requires an existing 'minishift' VM
+    """
+
+Scenario: User can start Minishift with specific Asia/Kolkata timezone
+  Given Minishift has state "Does Not Exist"
+    And image caching is disabled
+   When executing "minishift start --timezone Asia/Kolkata" succeeds
+   Then executing "minishift timezone" succeeds
+    And stdout should contain
```

Comment:
Would be nice here to also check from inside of the VM whether the zone change is reflected in the OS.

Reply:
It will be a dynamic check hence faction of seconds matters a lot. So It is difficult to verify the VM time with the specified timezone precisely. For example suppose VM local time : ```16:30:21``` is captured for ```Asia/Kolkata``` timezone through the code by using ```minishift timezone``` and in the very next line current time of the VM is captured through ```minishift ssh -- date```. There might be possibility that arises when current time inside the VM will be ```16:30:22``` and same will be applicable for minuets and hrs too, although it is a rare occurrence. 

However for verification propose we can directly use ```re``` in the feature file on ```minishift ssh -- date``` to match substring like IST, CEST and UTC

Reply:
I would suggest that we should check if `date` command inside Minishift has `IST` string or not. That should be enough IMO.

Reply:
Right if we do not care about the day, month, year..